README.md

```diff
@@ -49,9 +49,9 @@ Options:
 Latest Release: [https://github.com/mayswind/AriaNg-Native/releases](https://github.com/mayswind/AriaNg-Native/releases)
 
 > **For mac with Apple silicon**  
-> Before starting to install, you need to execute the following command, otherwise the system will show you the file is damaged.
+> Before starting to install, you need to execute the following command, otherwise the system will show you the file is damaged. You should not open the .dmg file and if you did already, eject it before running the command.
```

Comment:
I think the previous tips are clear, and the new content is a bit redundant. By the way, if you think the current tips don't mention executing the command on the downloaded installer file, you can modify the tips instead of adding asterisks to the command.

Reply:
<img width="830" height="607" alt="image" src="https://github.com/user-attachments/assets/1364d4f5-a1bd-40e9-a126-38a906c58fa8" />

is it not clear?